content/judgment.tex

```diff
@@ -43,9 +49,9 @@ \subsection{GameController Operator}
 They should count aloud the remaining seconds in a half once the time remaining is \qty{5}{\second} or less.
 Finally, they should repeat the calls of the head referee to make sure it was heard correctly.
 
-\subsection{Referee -- Team communication}
+\subsection{Referee — Team communication}
```

Comment:
I think this should be `Referee--Team Communication`, but I can fix this in a later commit.